src/ui/RelevanceInspector/TableBuilder.ts

```diff
@@ -0,0 +1,216 @@
+import { loadAgGridLibrary } from './AgGrid';
+import { first, map, each, flatten, find } from 'underscore';
```

Comment:
Is there a technical reason for which you are importing underscore modules this way? Otherwise I would prefer consistency across the code base.

Reply:
If we ever want to be able to use tree shaking for underscore, we'll need to use that syntax.

Reply:
Hmm, to this point, when I was playing the bundle analyzer to check why the Interface Editor Toolbox component was so large, I saw no different in final bundle size whether using `import *` or `import {method1, method2}`. This was with using uglifyjs which I am pretty sure was applying tree-shaking, but I may be wrong.

Reply:
Tree shaking is only available in webpack 4 and probably needs to be enabled.